src/reader/reader.js

```diff
@@ -17,6 +17,18 @@ class Reader extends Component {
     this.state = {ready: false}
   }
 
+  componentDidMount() {
+    const ua = window.navigator.userAgent
+    if (ua.indexOf("iPhone") > -1 || ua.indexOf("iPad") > -1) {
```

Comment:
Could remove this check if we want to apply it to all touch screens. https://developer.mozilla.org/en-US/docs/Web/API/Element/touchstart_event